cli/main.c

```diff
@@ -157,33 +144,65 @@ int main(int argc, char** argv) {
   // FIXME: this is temp till arg parse implemented.
   PkResult result;
 
-  if (argc == 1) {
+  static const char* const usage[] = {
+    "pocket [option] ... [-c cmd | file | -] [arg] ...",
+    NULL,
+  };
+
+  int debug = 0;
+  int quiet = 0;
+  int version = 0;
```

Comment:
Use `bool` here, even though bools are defined with ints in C, it's easier to read and understand the purpose of the variable.

```c
bool debug = false, help = false, quiet = false, version = false;
```